data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+ACME_BASE=/var/lib/acme
+mkdir -p ${ACME_BASE}/acme/private
+
+restart_containers(){
+	for container in $*; do
+		curl -X POST \
+			--unix-socket /var/run/docker.sock \
+			"http:/containers/${container}/restart"
```

Comment:
Well, I'm currently thinking about this. But I'm sure, that the `containers`-bit is part of the filename in the URL and not some server name. And a possible explanation is that the `//example.com` in an URL is an optional hostname. Because we're using `--unix-socket` to connect to docker, there is no hostname needed and therefore we're using the "default" hostname.

If this is true. That 'd be awesome.